Whitepaper: more discussion on CSIRTs (#208)
orcwg/orcwg: cyber-resilience-sig/whitepapers/stewards-and-cra.md

- the highest number of users of products of that Steward are located

In practice, we can assume that the Steward/Project need to report by default to the CSIRT of the country the Steward has its headquarters.

If the Steward has no European official representation, we assume the reporting can be done to one of the CSIRTs of big EU countries (like Germany, France, Spain, Italy etc),
except if the specific Project has a strong user base in another country[^CSIRTload]. This subject should be clarified, especially for Stewards and Projects
that do not have any representation in EU and for cases when there might be a language barier between the CSIRT and the Project. Also, ENISA

> could be a choice of a "CSIRT of last resoirt" for Stewards without a clear national alignment.

Reviewer: ```suggestion
could be a choice of a "CSIRT of last resort" for Stewards without a clear national alignment.
```

[^pwdnote]: This could lead, however, to high load on those CIRTs. The CIRT network may want to introduce a load-balancing mechanism
to solve this issue. If that is the case, it would be a good practice to notify affected Stewards and Projects which CSIRT will be responsible
for a given Project.

If the Steward/Project could not figure out the appropriate CSIRT using those rules, we currently assume that they can choose the one that they find the most appropriate.

> As the legislation seems to assume one CSIRT for Steward, multiple Projects will need to share the decision on the choice of the CSIRT. It would be the best

Reviewer: ```suggestion
As the legislation seems to assume a single CSIRT per Steward, multiple Projects will need to share the decision on the choice of the CSIRT. It would be the best
```

practice to take that decision together between all concerned Projects and the Steward representatives.

It will be also a good practice to document which CSIRT is the main one for each Project in the standard security documentation for that Project so it is easy

> to find both for anyone who wishes to report a vulnerability, and for contributors to that Project.

Author: Included, thanks!

- the highest number of users of products of that Steward are located

In practice, we can assume that the Steward/Project need to report by default to the CSIRT of the country the Steward has its headquarters.

If the Steward has no European official representation, we assume the reporting can be done to one of the CSIRTs of big EU countries (like Germany, France, Spain, Italy etc),
except if the specific Project has a strong user base in another country[^CSIRTload]. This subject should be clarified, especially for Stewards and Projects
that do not have any representation in EU and for cases when there might be a language barier between the CSIRT and the Project. Also, ENISA
could be a choice of a "CSIRT of last resort" for Stewards without a clear national alignment.

[^pwdnote]: This could lead, however, to high load on those CSIRTs. The CSIRT network may want to introduce a load-balancing mechanism
to solve this issue. If that is the case, it would be a good practice to notify affected Stewards and Projects which CSIRT will be responsible
for a given Project.

> If the Steward/Project could not figure out the appropriate CSIRT using those rules, we currently assume that they can choose the one that they find the most appropriate.

Reviewer 2: I would avoid this  - i.e. let the OSS 'shop'  around. Or play geo/tax games.

Much easier if it is ENISA as a top level CIRT - and/or give ENISA or the CIRT network the option to inform an OSS steward as to what the 'home CIRT' is for them.

As the legislation seems to assume a single CSIRT per Steward, multiple Projects will need to share the decision on the choice of the CSIRT.

It will be also good practice to explicitly name the main CSIRT for each Project in its standard security documentation, 
making it easily findable for anyone wishing to report a vulnerability, as well as for contributors to that Project.

Open Source usage being global in nature, different CSIRTs will likely receive reports about vulnerabilities for the same Project. Moreover,
two parties may interpret conditions differently, and as such report to different CSIRTS for the same Project. It will be necessary to define rules of how

> those reports will be routed. ENISA may play a coordinator role in this scenario.

Reviewer 2: I would reword this - and explain that we want to avoid it - with reference to the above - you stick to a CIRT, you use ENISA or you get a CIRT assigned by ENISA/the-CIRT-network
